### PR TITLE
Landing Page and Responsiveness init ; more improvements

### LINKED ISSUE #330: Landing Page Layout

### PR DESCRIPTION
**Pull request by roienatan**
resolves: #330

- Responsiveness init
- Global scss variables

### CODE REVIEW

File: packages/web/src/containers/App/App.tsx [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+  //const is_authorized = useSelector(authentificated());
 
   return (
     <div className="App">
       <Header />
       <Content>
         <Switch>
           <Route path={ROUTE_PATHS.AUTH} component={AuthContainer} />
-          <PrivateRoute path="/" exact component={Dashboard} authentificated={is_authorized} />
+          {/* <PrivateRoute path="/" exact component={Dashboard} authentificated={is_authorized} /> */}
```
Comment (razoorka) [inline packages/web/src/containers/App/App.tsx]: Can be removed

Comment (roienatan) [reply packages/web/src/containers/App/App.tsx]: Ok it's just the init and it's still marked as draft, I'll make many more changes 👍

File: packages/web/src/containers/Landing/components/LandingContainer/ContactUs/ContactUs.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [description, setDescription] = useState("");
+
+  return (
+    <div className="contact-us-wrapper">
+      <div className="title">
+        <h1>Contact us</h1>
+        <div className="email-wrapper">
+          <img src={require("../../../../../shared/assets/icons/email.svg")} alt="envelope" />
```
Comment (razoorka) [inline packages/web/src/containers/Landing/components/LandingContainer/ContactUs/ContactUs.tsx]: src="/icons/email.svg"

File: packages/web/src/containers/Landing/containers/LandingContainer/LandingContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  const screenSize = useSelector(getScreenSize());
+
+  return (
+    <div className="landing-wrapper">
+      <section className="landing-top">
+        <div className="info">
+          <img src="icons/logo-white.svg" alt="logo" width="150px" />
+          <h1>
+            Launch Collective Action. <br />
+            <span style={{ color: "#ff5e9d" }}>Catalyze a movement</span>
```
Comment (razoorka) [inline packages/web/src/containers/Landing/containers/LandingContainer/LandingContainer.tsx]: Can we do not use inline styles?

File: packages/web/src/containers/Landing/containers/LandingContainer/LandingContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+          <p>
+            Humanity's greatest achievements were made by massive <br /> collaborations. Common empowers groups to
+            collaborate with no <br /> single organizer. And when every member gets an equal part in <br /> every step
+            of the process, we enter a new era of trust and <br /> collaboration.
+          </p>
+        </div>
+      </section>
+      <section>
+        <h1>Featured Commons</h1>
+        <div style={{ marginBottom: "30px" }}>Browse some of the emerging groups on the Common app</div>
```
Comment (razoorka) [inline packages/web/src/containers/Landing/containers/LandingContainer/LandingContainer.tsx]: same as above about styles

File: packages/web/src/containers/App/App.tsx [hunk 5]
```diff
@@ -1,29 +1,33 @@
...
+import { LandingContainer } from "../Landing";
 import { ROUTE_PATHS } from "../../shared/constants";
+import { ScreenSize, SMALL_SCREEN_BREAKPOINT } from "./constants";
+import { useDispatch } from "react-redux";
+import { changeScreenSize } from "../../shared/store/actions";
 
 const App = () => {
-  const is_authorized = useSelector(authentificated());
+  const dispatch = useDispatch();
+  const screenSize = window.matchMedia(`(min-width: ${SMALL_SCREEN_BREAKPOINT})`);
```
Comment (razoorka) [inline packages/web/src/containers/App/App.tsx:15]: you can place screen size as a prop to <Header /> instead of calling selector of screen size inside

**Review by razoorka — APPROVED** [review]

**Pull request merged**